.github/workflows/ci.yml

```diff
@@ -26,12 +26,12 @@ jobs:
         with:
           php-version: ${{ matrix.php-version }}
       - name: Remove dev tools to not interfere with dependencies
-        run: composer remove --dev friendsofphp/php-cs-fixer phpstan/phpstan-phpunit phpstan/phpstan
+        run: composer remove --dev friendsofphp/php-cs-fixer phpstan/phpstan-phpunit phpstan/phpstan rector/rector
       - name: Composer cache
         uses: actions/cache@v3
         with:
           path: ${{ env.HOME }}/.composer/cache
-          key: ${{ runner.os }}-php-${{ hashFiles('**/composer.lock') }}
+          key: ${{ runner.os }}-php-${{ hashFiles('**/composer.json') }}
```

Comment:
oh. we don't even have a checked in lock file, right?

Reply:
exactly, that's why the cache was not properly used yet :)